examples/scaling_example/scaling.c

```diff
@@ -289,6 +292,19 @@ run_advanced_rings(struct onvm_nf_info *nf_info) {
         }
 
         while (keep_running && rx_ring && tx_ring && nf) {
+                /* Check for a stop message from the manager. */
+                if (unlikely(rte_ring_count(msg_q) > 0)) {
+                        msg = NULL;
+                        rte_ring_dequeue(msg_q, (void **)(&msg));
+                        if (msg->msg_type == MSG_STOP) {
+                                keep_running = 0;
+                        }
+                        else {
+                                printf("Received message %d, ignoring", msg->msg_type);
+                        }
+                        rte_mempool_put(rte_mempool_lookup(_NF_MSG_POOL_NAME), (void *)msg);
```

Comment:
Save the output of rte_mempool_lookup(_NF_MSG_POOL_NAME) to a variable, looking it up every time isn't a good coding practice